docs/docs-beta/docs/dagster-plus/deployment/deployment-types/hybrid/architecture.md

```diff
@@ -35,27 +23,35 @@ The agent polls the agent API to see if any work needs to be done and launches u
 
 All user code runs within your environment, in isolation from Dagster system code.
 
----
-
 ## The agent
 
 Because the agent communicates with the Dagster+ control plane over the agent API, it's possible to support agents that operate in arbitrary compute environments.
 
 This means that over time, Dagster+'s support for different user deployment environments will expand and custom agents can take advantage of bespoke compute environments such as HPC.
 
-Refer to the [Agents documentation](/todo) for more info, including the agents that are currently supported.
-
----
+See the [setup page](index.md#dagster-hybrid-agents) for a list of agents that are currently supported.
 
 ## Security
 
-This section describes how Dagster+ interacts with user code. To summarize:
+Dagster+ Hybrid relies on a shared security model.
+
+The Dagster+ control plane is SOC 2 Type II certified and follows best practices such as:
+- encrypting data at rest (AES 256) and in transit (TLS 1.2+)
+- highly available, with disaster recovery and backup strategies
+- only manages metadata such as pipeline names, execution status, and run duration
+
+The execution environment is managed by the customer:
+- Dagster+ doesn't have access to user code—your code never leaves your environment. Metadata about the code is fetched over constrained APIs.
+- All connections to databases, file systems, and other resources are made from your environment.
+- The execution environment only requires egress access to Dagster+. No ingress is required from Dagster+ to user environments.
+
+Additionally, the Dagster+ agent is [open source and auditable](https://github.com/dagster-io/dagster-cloud)
 
-- No ingress is required from Dagster+ to user environments
-- Dagster+ doesn't have access to user code. Metadata about the code is fetched over constrained APIs.
-- The Dagster+ agent is [open source and auditable](https://github.com/dagster-io/dagster-cloud)
+Common security considerations in Dagster+ hybrid include:
```

Comment:
Are these recommendations for the user to follow? If so, I'll move them to the index page for the hybrid section (which is basically the setup page).

Reply:
Yes they are